dashboard/app/controllers/level_starter_assets_controller.rb

```diff
@@ -14,6 +15,9 @@ def show
       summarize(file_obj, friendly_name, uuid_name)
     end.compact
 
+    # Setting custom header here allows us to access the csrf-token and manually use for destroy method.
+    headers['csrf-token'] = form_authenticity_token
```

Comment:
I think this can this be removed, now that we're skipping the authenticity check for this route